spec/models/open_studios_event_spec.rb

```diff
@@ -81,6 +81,35 @@
     end
   end
 
+  describe 'lifecycle' do
+    describe 'before_save' do
+      it 'generates time slots if special event fields have changed' do
+        now = Time.zone.today
+        later = now + 1.day
```

Comment:
I think this test will break when `today` is the last day of the month unless timecop (or whatever Ruby's new-ish baked in testing time management) is configured. Refer to my comment [here](https://github.com/bunnymatic/mau/pull/137/files#diff-5a2ba045c845b28a8c804f41918c6e6c6ceec7fc56474ff398bb70fe9ef8ec18R55).